Source/Layout/ASYogaLayoutSpec.h

```diff
@@ -0,0 +1,21 @@
+//
+//  ASYogaLayoutSpec.h
+//  AsyncDisplayKit
+//
+//  Created by Scott Goodson on 5/6/17.
+//  Copyright © 2017 Facebook. All rights reserved.
+//
+
+#import <AsyncDisplayKit/ASAvailability.h>
+
+#if !YOGA_TREE_CONTIGUOUS /* !YOGA_TREE_CONTIGUOUS */
+
+#import <AsyncDisplayKit/ASDisplayNode.h>
+#import <AsyncDisplayKit/ASLayoutSpec.h>
+
+@interface ASYogaLayoutSpec : ASLayoutSpec
+@property (nonatomic, strong, nonnull) ASDisplayNode *rootNode;
+@property (nonatomic, strong, nullable) NSArray<ASDisplayNode *> *yogaChildren;
+@end
+
+#endif /* !YOGA_TREE_CONTIGUOUS */
```

Comment:
It seems like this spec only accept display nodes. Any particular reasons why it can't just work with generic ASLayoutElement instances? It would be nice if it can do that, and even have the same API with `ASStackLayoutSpec`, should we need to make it the default stack algorithm spec and ensure the transition is seamless. Plus, we can leverage existing snapshot tests of `ASStackLayoutSpec` to test this Yoga spec.